backend/s3/s3.go

```diff
@@ -173,6 +173,9 @@ var providerOption = fs.Option{
 	}, {
 		Value: "Selectel",
 		Help:  "Selectel Object Storage",
+	}, {
+		Value: "SpectraLogic",
+		Help:  "SpectraLogic BlackPearl",
```

Comment:
This should probably be "Spectra Logic Black Pearl" as this message is shown to the user